Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -48,7 +52,7 @@ There are usually two release engineers working at any given time. It goes witho
 <br/>	*It starts after all opened issues had been adressed and can take several cycles until QA's approval*
```

Comment:
Can we mention who is responsible for creating the release JIRA board? (in the past was the release engineer responsibility)

Reply:
According to [the more general doc in Confluence](https://babylonpartners.atlassian.net/wiki/spaces/V4/pages/767623303/iOS+Android+Release+process#iOS&AndroidReleaseprocess-JiraReleaseBoard) it's Lilla's responsibility – that's also why it's not listed here as part of the Release Engineer's tasks I guess

Reply:
So, you chatted with Lilla and in fact the confluence do is not up-to-date, but it's still not the Release Engineer's role to update it.

> [Lilla] The release board does exist, it’s just the filter that needs to be changed. Which is done by @JasonDobo